packages/@okta/vuepress-site/docs/guides/sign-in-to-spa-embedded-widget/main/react/basic-sign-in.md

```diff
@@ -0,0 +1,202 @@
+### Create routes
+
+Typically, an app contain routes that require authentication in order to render. Use the `SecureRoute` component from [Okta React SDK](https://github.com/okta/okta-react) (`@okta/okta-react`) to define authenticated routes for your app. The following are some basic routes that you need to configure for your app:
```

Comment:
In my React SPA guide, I used `< SecureRoute >` instead of `SecureRoute `. I'm not sure which is correct.

It seems strange to say Okta React SDK and @okta/okta-react in the same sentence.

Reply:
Thanks, I'll use the angle brackets too. And remove the second `@okta/okta-react` reference.

Reply:
```suggestion
Typically, an app contain routes that require authentication in order to render. Use the `<SecureRoute>` component from [Okta React SDK](https://github.com/okta/okta-react) to define authenticated routes for your app. The following are some basic routes that you need to configure for your app:
```